search/tests/tests.py

```diff
@@ -625,6 +684,46 @@ def test_datetime_conversion(self):
         json_datetime = "2015-01-31T07:30:28.65785Z"
         self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28, 65785))
 
+    def test_timezone_conversion(self):
```

Comment:
@martynjames suggestion: this test doesn't test important edge cases:

* Both have timezones, represent the same moment in time, but timezones are different - should be equal
* Both have timezones, represent different moments in time, local time is equal but timezones are different - should not be equal